server/base/src/test/java/org/apache/accumulo/server/util/FileSystemMonitorTest.java

```diff
@@ -37,7 +37,7 @@
 
   @Test
   public void testFilteredMountEntries() throws Exception {
-    String[] mountEntries = new String[] {"rootfs / rootfs rw 0 0",
+    String[] mountEntries = {"rootfs / rootfs rw 0 0",
```

Comment:
Thoughts: I really wonder about the portability of this FileSystemMonitor. It seems dubious. I'm not even sure if it works properly with systemd or recognizes m.2 and nvme drives. The FileSystemMonitor seems more like a monitoring utility which should exist separate from Accumulo.